.env.example

```diff
@@ -19,13 +20,54 @@ CHRONOS_SCHEDULE_JOBS='0 */10 * * * *'
 CHRONOS_UPDATE_TOKENS='0 0 0 1,15 * *'
 CHRONOS_PROCESS_RECURRENCE='0 */5 * * * *'
 CHRONOS_AUTOPAUSE='0 0 5 * * *'
-CI='true'
+# Enter a short url redirect service for invitations, it needs to redirecto to /invitation-link
+# INVITATION_SHORTLINK='example.com'
+
+# AUTHENTICATION
+AUTH_INTERNAL_DISABLED='false'
+AUTH_GOOGLE_DISABLED='false'
+AUTH_SSO_DISABLED='false'
+OAUTH2_REDIRECT=''
+
+# MONITORING
```

Comment:
I would replace this by `# Datadog`, as it is very specific, and have the following vars:
```
DD_API_KEY=''
DD_APPLICATIONID=''
DD_APPSEC_ENABLED='true'
DD_CLIENTTOKEN=''
DD_ENV='ENVIRONMENT'
DD_LOGS_INJECTION='true'
DD_RUNTIME_METRICS_ENABLED='true'
DD_SERVICE='DD_SERVICE_NAME'
DD_TRACE_AGENT_URL='PROTOCOL://URL:PORT'
DD_TRACE_ENABLED='true'
```